src/components/association-editor/AssociationImageLink.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+
+  <document-link :document="image" class="has-hover-background" target="_blank">
+    <figure class="image is-96x96 association-image-link-child">
+      <img :src="getUrl(image)" :title="image.locales[0].title">
+    </figure>
+    <document-title :document="image" class="association-image-link-child"/>
```

Comment:
How do you handle very long image titles (which may happen). Maybe add an ellipsis when title is too long. See e.g. `max300chars` filter.

Reply:
Add in the last commit.
On Iphone 5 with 50 characters. Maybe we could go until 60 chars. but depends on the case and typo.
![image](https://user-images.githubusercontent.com/24532066/56378727-2917ca80-620e-11e9-98ac-9835138881be.png)